tests/integration/profiling/rule_based_profilers/test_profiler_user_workflows.py

```diff
@@ -275,7 +275,7 @@ def test_alice_profiler_user_workflow_single_batch(
         == alice_columnar_table_single_batch["expected_expectation_suite"]
     )
 
-    assert mock_emit.call_count == 43
+    assert mock_emit.call_count == 54
```

Comment:
Calling `get_batch()` again each of Domains because we are checking that the `len()` == 1.